metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -77,29 +87,80 @@ final class Embedded(
     AbsolutePath(out)
   }
 
+  private val presentationCompilers: TrieMap[String, URLClassLoader] =
```

Comment:
out of curiosity: why not move the classloading inside the pc-interfaces, similarly to what happens in Scalafix/Scalafmt?

Reply:
The main reason is because I like making JDK `ServiceLoader` the "public abstraction", users of this API should know there is reflection going on.

- scalafmt handles downloading via coursier, so it's a bit different
- I didn't know about `ServiceLoader` when I wrote scalafix-interfaces